convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -343,3 +358,136 @@ def test_backup_package_file_complete(
                     assert f.read() == "Content for testing of file %s" % original_file_path
             elif status == "missing":
                 assert not os.path.isfile(original_file_path)
+
+
+class TestBackupRepository:
+    @all_systems
+    def test_backup_repository_complete(
+        self, monkeypatch, tmpdir, backup_repository_action, global_backup_control, pretend_os
+    ):
+        """Test backup, remove the originals and restore them from backup."""
+        yum_repo = generate_repo(tmpdir, name="test.repo")
+
+        backup_dir = str(tmpdir.mkdir("backup"))
+
+        monkeypatch.setattr(backup_system, "DEFAULT_YUM_REPOFILE_DIR", os.path.dirname(yum_repo))
+        monkeypatch.setattr(files, "BACKUP_DIR", backup_dir)
+
+        backup_repository = backup_repository_action
+
+        backup_repository.run()
+
+        # Remove the original files
+        os.remove(yum_repo)
+        assert not os.path.isfile(yum_repo)
+
+        global_backup_control.pop_all()
+
+        # Check presence of restored files
+        assert os.path.isfile(yum_repo)
+        with open(yum_repo, mode="r") as f:
+            assert f.read() == os.path.basename(yum_repo)
+
+    @all_systems
+    def test_backup_repository_redhat(
+        self, monkeypatch, tmpdir, backup_repository_action, global_backup_control, pretend_os, caplog
```

Comment:
```suggestion
        self, monkeypatch, tmpdir, backup_repository_action, pretend_os, caplog
```

Reply:
Yes, in this test is nothing backed up, so this is not necessary/ I'm removing it. Thanks